patterns/hidden-post-navigation.php

```diff
@@ -0,0 +1,14 @@
+<?php
+/**
+ * Title: Post navigation
+ * slug: twentytwentyfour/post-navigation
```

Comment:
should this be a hidden one?

Reply:
The other hidden patterns look the same, it says `hidden` in the file name and `inserter: no` below, so it's looking good to me?

Reply:
forgive my blindness and forget my comment :D